beta/nncf/tensorflow/quantization/algorithm.py

```diff
@@ -247,5 +251,15 @@ def loss(self) -> CompressionLoss:
     def initialize(self, dataset=None, loss=None):
         self._initializer(self._model, dataset, loss)
 
+        init_bn_adapt_config = self._config.get('initializer', {}).get('batchnorm_adaptation', {})
+        if init_bn_adapt_config:
```

Comment:
Is there another way to check that we need to run BA?

Reply:
Implemented check is aligned with the docs:
`It [BN adaptation] can be enabled by setting a non-zero value of num_bn_adaptation_samples in the batchnorm_adaptation section of the initializer configuration`.

From the user perspective, this is a clear check I guess. Which other options could be applied here?